x/participationrewards/keeper/hooks.go

```diff
@@ -146,6 +149,105 @@ func (k *Keeper) AfterZoneCreated(ctx sdk.Context, zone *icstypes.Zone) error {
 		Data: localDenomBytes,
 	})
 
+	// Create LiquidAllowedDenomProtocolData for the host zone
+	hostZoneDenom := types.LiquidAllowedDenomProtocolData{
+		ChainID:               zone.ChainId,
+		RegisteredZoneChainID: zone.ChainId,
+		IbcDenom:              zone.BaseDenom,
```

Comment:
This isn't correct; the ibc denom will be an IBC denom for the host zone, because it originates from the local zone.